### ISSUE TITLE
Whiteboard integration fails to start on CentOS7 because of libstdc++ version dependency

### ISSUE THREAD

**Issue by Gummikavalier** [nextcloud/integration_whiteboard#24]
## Description

When trying to use whiteboard, user gets the error:
`Impossible to load Spacedeck whiteboard "cURL error 7: Failed connect to localhost:9666; Connection refused (see https:\/\/curl.haxx.se\/libcurl\/c\/libcurl-errors.html) for http:\/\/localhost:9666\/api\/spaces"`

In closer inspection Whiteboard/Spacedeck is not running at all on CentOS7 servers. Ubuntu 20.04 LTS installations don't suffer from this.

Integration fails to start on CentOS7 because of missing libstdc++ version dependency:
```
spacedeck.nexe.bin: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /var/www/html/nextcloud/apps/integration_whiteboard/data/spacedeck/spacedeck.nexe.bin)
spacedeck.nexe.bin: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /var/www/html/nextcloud/apps/integration_whiteboard/data/spacedeck/spacedeck.nexe.bin)
spacedeck.nexe.bin: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /var/www/html/nextcloud/apps/integration_whiteboard/data/spacedeck/spacedeck.nexe.bin)
```

I haven't found a good way to fill out the dependency apart from compiling from source, which is not very maintainable scenario.

**Comment by IzeRecord** [comment]
Have the same problem. Did you find a solution? Any Idea to solve this?

**Comment by Gummikavalier** [comment]
No. It would be resolvable via installing the missing modules by hand into the OS by building them from the source, but that'd not be very maintainable situation for the long run in case they needed updates.
